svf-llvm/tools/AE/ae.cpp

```diff
@@ -1,16 +1,44 @@
+//===- ae.cpp -- Abstract Execution -------------------------------------//
 //
-// Created by Jiawei Ren on 11/28/23.
+//                     SVF: Static Value-Flow Analysis
 //
-#include "AbstractExecution/IntervalExeState.h"
-#include "AbstractExecution/RelExeState.h"
-#include "AbstractExecution/RelationSolver.h"
-#include "SVF-LLVM/LLVMUtil.h"
-#include "Util/Z3Expr.h"
+// Copyright (C) <2013-2017>  <Yulei Sui>
+//
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+//
+//===-----------------------------------------------------------------------===//
+
+/*
+ // Abstract Execution
+ //
+ // Author: Jiawei Wang, Xiao Cheng, Jiawei Yang, Jiawei Ren, Yulei Sui
+ */
+#include "SVF-LLVM/SVFIRBuilder.h"
+#include "WPA/WPAPass.h"
+#include "Util/CommandLine.h"
 #include "Util/Options.h"
-#include <chrono>
 
+#include "AbstractExecution/AE-SVFIR/BufOverflowChecker.h"
+#include "AbstractExecution/AE-Core/RelExeState.h"
+#include "AbstractExecution/AE-Core/RelationSolver.h"
+
+
+using namespace llvm;
+using namespace std;
 using namespace SVF;
-using namespace SVFUtil;
```

Comment:
why add this?

Reply:
Thanks
I remove 'using namespace llvm'
and remain the SVFUtil in latest commit